bundle/libraries/local_path.go

```diff
@@ -29,21 +27,49 @@ func IsLocalPath(p string) bool {
 		return true
 	}
 
-	// If the path has another scheme, it's a remote path.
-	if isRemoteStorageScheme(p) {
+	if IsRemotePath(p) {
 		return false
 	}
 
-	// If path starts with /, it's a remote absolute path
+	// If the path is absolute, it's a remote path.
 	return !path.IsAbs(p)
 }
 
-// IsEnvironmentDependencyLocal returns true if the specified dependency
+func IsRemotePath(p string) bool {
+	if isRemoteStorageScheme(p) {
+		return true
+	}
+
+	// if the path is not absolute, it's not a remote path
+	if !path.IsAbs(p) {
+		return false
+	}
+
+	// If the path is absolute, it's might be a remote path.
+	possiblePrefixes := []string{
+		"/Workspace",
+		"/Users",
+		"/Volumes",
+		"/Shared",
+		"/mnt",
+		"/dbfs",
+	}
+
+	for _, prefix := range possiblePrefixes {
+		if strings.HasPrefix(p, prefix) {
+			return true
```

Comment:
I still don't understand why we need these prefixes.

The heuristic was: if a path is absolute, we assume it's remote.

The heuristic becomes: if a path is absolute but it doesn't have any of these prefixes, it's not considered remote.

This means that for a path like `/foo/bar` both `IsLocalPath` and `IsRemotePath` return false.

Reply:
Fixed in the latest revision